src/mmw/mmw/settings/data/scripts/bufferDRWI.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+var turfBuffer = require('turf-buffer'),
+    fs = require('fs');
+    drwiPath = '../drwi_perimeter.json',
+    drwiBufPath = '../drwi_buffered_perimeter.json',
+    drwi = require(drwiPath),
+    distance = 3,
+    unit = 'miles',
+    drwiBuf = turfBuffer(drwi, distance, unit);
+
```

Comment:
Can this be combined into a general purpose buffer script with DRB or DRWI flag arguments to set the path variables or is it better to have this as a separate script?

Reply:
It probably could be parameterized into a single script, but since different shapes could have different buffer radii based on their size and shape, and there's little else in these files but those variables, I decided to keep them separate.